scripts/microbe_stage/organelle_table.as

```diff
@@ -255,522 +207,124 @@ OrganelleComponentFactory@ pilusOrganelleFactory()
         "Pilus");
 }
 
-// ------------------------------------ //
-// Sets up the organelle table
+//! This replaced the old tables that specified things for cells.
+//! This is clearer as to what are valid properties
+class OrganelleParameters{
```

Comment:
I feel like this stuff should have been pulled to C++. In fact the entire organelle_table.as contents should be instead in C++. So that the scripts would call a C++ registered function. In scripts the code would be `getThriveGame().getOrganelle("organellename");`
And then instead PlacedOrganelle would do whatever it needs with Organelle.
Basically so that the json files would (almost) fully configure the organelle definitions with C++ code there to load it and provide the info to scripts.